examples/hello_world/gcp/BUILD

```diff
@@ -0,0 +1,38 @@
+#
+# Copyright 2020 The Project Oak Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+load("@graknlabs_bazel_distribution//packer:rules.bzl", "assemble_packer", "deploy_packer")
+load("//oak/common:app_config.bzl", "serialized_config")
+
+package(
+    licenses = ["notice"],
+)
+
+exports_files(["config.json"])
+
+assemble_packer(
+    name = "assemble-oak-gcp-hello-world",
+    config = ":config.json",
+    files = {
+        "//oak/server/dev:dev_oak_runner" : "dev_oak_runner",
+        "//examples/hello_world/config:config.bin" : "config.bin",
```

Comment:
As #474 is now merged, this has now been updated to use /examples/hello_world/config:config rather than having a local application config.